src/lib.rs

```diff
@@ -1856,8 +1883,7 @@ impl Iterator for ChunkRawIter<'_> {
 
 impl iter::FusedIterator for ChunkRawIter<'_> {}
 
-#[inline(never)]
-#[cold]
+#[inline]
```

Comment:
This should never be inlined, the whole point is that it is a very cold path, that is out of lined. Function call overhead is negligible compared to the cost of panicking, and inlining this code will slow down the vast majority common case when there is no OOM.

Reply:
`panic!()` expands to a call to `core::panicking::panic`, which is itself marked `#[inline(never)]`